pkg/config/config.go

```diff
@@ -255,6 +255,7 @@ func addValue(cg *cb.ConfigGroup, value *standardConfigValue, modPolicy string)
 }
 
 // addOrdererPolicies adds *cb.ConfigPolicies to the passed Orderer *cb.ConfigGroup's Policies map.
+// It checks that the BlockValidation policy is defined alongside the standard policy checks.
 func addOrdererPolicies(cg *cb.ConfigGroup, policyMap map[string]*Policy, modPolicy string) error {
```

Comment:
nit: Given this is specific to the orderer, we could move this to `pkg/config/orderer.go`